src/sign_up/sign_up.jsx

```diff
@@ -1,24 +1,41 @@
-import React from "react";
+import React, { useState } from "react";
 import "./sign_up.css";
 
-class SignUp extends React.Component {
-  constructor(props) {
-    super(props);
+const SignUp = () => {
+  const [email, setEmail] = useState("");
```

Comment:
inconsistent quotes again, check your whole project. you can also add `eslint` rule so it will be checked automatically. you can also set up your IDE for this